### ISSUE TITLE
Clearer discussion posts

### ISSUE THREAD

**Issue by simplybenuk** [nhsengland/futurenhs#351]
Original discussion post needs to be visually clearer to see. As the comments and replies stand out more, users are drawn to that part of a forum discussion initially

![image](https://user-images.githubusercontent.com/9471595/189076177-0294a4da-e877-4631-818e-3baafc331722.png)

**Comment by tim-hunt** [comment]
Agreed, do you have an example in mind or are you ok with us trying some things and seeing what looks best to us?

**Comment by tim-hunt** [comment]
Having a look around in the NHS Front end toolkit theres a few components that could work?

https://nhsuk.github.io/nhsuk-frontend/components/card/basic-card.html
![image](https://user-images.githubusercontent.com/108523116/189316529-865029e1-9b05-4b9b-82b0-f6b24c1f8c62.png)

https://nhsuk.github.io/nhsuk-frontend/components/card/feature-card.html
![image](https://user-images.githubusercontent.com/108523116/189316029-1b8a393d-160c-4f98-8fc4-99e902a094f1.png)

https://nhsuk.github.io/nhsuk-frontend/components/hero/hero-image-content.html
![image](https://user-images.githubusercontent.com/108523116/189319128-d0d7ee02-2223-4932-9aac-ac036064a712.png)

**Comment by simplybenuk** [comment]
Like these options, I think there is a design mock up of one that looks just like one of these. @marianthi-design could you add in the UX one to this issue?

**Comment by marianthi-design** [comment]
Hi @tim-hunt, thanks for sharing these.

@Richard-CDS did a design on this:
<img width="1861" alt="Files and forums" src="https://user-images.githubusercontent.com/84837658/189602043-4c9d9c73-edd0-44db-ac1e-6c641ad1ce84.png">

It looks more like the first component you posted.

**Comment by marianthi-design** [comment]
Thanks @tim-hunt !
If we could move the blue line at the bottom great, if not let's leave it without a line on the left. Just a plain card. So that it looks different visually to the replies.
